### PR TITLE
Update colors and add logo asset

### PR DESCRIPTION
**Pull request by codehag**

### CODE REVIEW

File: assets/css/style.scss [hunk 1]
```diff
@@ -16,7 +16,7 @@ a {
 
 .tc39-box {
   border: transparent 1rem solid;
-  background-color: #ff9900;
+  background-color: #FA7C23;
```
Comment (xtuc) [inline assets/css/style.scss]: `#fc7c00` is the official color of the logo? (according to the svg)

Comment (codehag) [reply assets/css/style.scss]: thanks!

Comment (littledan) [reply assets/css/style.scss]: I don't understand this `.tc39-box` class and div. Why don't we just reference the SVG file instead?

**Comment by codehag** [comment]
That is the plan! But the svg was added in a different pr so that will be
done outside of this one

**Pull request merged**